libs/synchronization/tests/unit/CMakeLists.txt

```diff
@@ -3,3 +3,96 @@
 # SPDX-License-Identifier: BSL-1.0
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set(tests
+  condition_variable
+  counting_semaphore
+  local_latch
+  local_barrier
+  local_barrier_count_up
+  local_barrier_reset
+  local_dataflow
+  local_dataflow_boost_small_vector
+  local_dataflow_executor
+  local_dataflow_std_array
```

Comment:
The dataflow tests should maybe not be in this module?

Reply:
Yeah I wasn't sure about those I'll leave them where they were !